extensions/amp-video/0.1/amp-video.js

```diff
@@ -168,6 +168,9 @@ export class AmpVideo extends AMP.BaseElement {
     /** @private {boolean} */
     this.isPlaying_ = false;
 
+    /** @private {boolean} */
+    this.isManagedByBitrate_ = this.hasAnyBitrateSources_();
```

Comment:
It's easier if it's cached, and avoids calculating it when in between mediapool operations, which could return the wrong value (ex: in between SwapIntoDom and UpdateSources)

Reply:
Okay, let's:

- move this work to the `buildCallback` (media pool will wait until build/layoutCallback resolves so it's safe)
- rename to something even more accurate about what it's used for "should be managed by the flexible bitrate class", rn it sounds like it's already managed and "bitrate" is a bit unclear/misleading

Reply:
Maybe also add a code comment in the buildCallback to say that we look at the sources here before the mediapool removes them